tests/pytests/functional/formulas/test_sudoers.py

```diff
@@ -0,0 +1,26 @@
+"""
+Tests using sudoers formula
+"""
+import pytest
+
+
+@pytest.fixture(scope="module")
+def _formula(saltstack_formula):
+    with saltstack_formula(name="sudoers-formula", tag="0.25.0") as formula:
+        yield formula
+
+
+@pytest.fixture(scope="module")
+def modules(loaders, _formula):
+    loaders.opts["file_roots"]["base"].append(
+        str(_formula.state_tree_path / f"{_formula.name}-{_formula.tag}")
+    )
+    return loaders.modules
+
+
+@pytest.mark.skip_on_windows
+@pytest.mark.destructive_test
+def test_sudoers_formula(modules):
+    ret = modules.state.sls("sudoers")
+    for staterun in ret:
```

Comment:
Can we add the extra checks here like the previous tests, because if we never loop through ret then we will get a false positive. Can you also assert the result is a bool (true/false)   

This also goes for the vim test

Reply:
Done

Reply:
I'm still not seeing extra checks before the for loop. We need to ensure we assert there are no failures before we go into the for loop or it will lead to false positives.